### ISSUE TITLE
Initial setup program

### ISSUE THREAD

**Issue by moondeck** [postmarketOS/pmbootstrap#1090]
Just thought of this, how about we implement some sort of small program (running on wayland/x11) that will let the user set up the phone, just like on Android/iOS/whatever? Connect to WiFi, create a user, choose and install the graphical environment. What do others think?

**Comment by ollieparanoid** [comment]
So basically it's a "netinstall" image of postmarketOS, where you download and install the GUI on the first run via wifi? Sounds useful to me 👍 

See also: #63 about a graphical installer (that does *not* run on the target device, it's more like a fancy version of pmbootstrap running on the installing PC).

**Comment by moondeck** [comment]
Yeah, i think that would be really awesome... i mean everyone has wifi nowadays (especially if you have a smartphone) and distributing images with each packaged GUI and for each device would be a major PITA. Self-building on the other hand would be a terrible idea for a possible end user.

**Comment by ollieparanoid** [comment]
Well, self-building could work as well with a graphical installer (fancy interface for pmbootstrap), see the mockups @MartijnBraam made in #63 and we could even do more hand-holding with flashing it onto the device.

But the net installation method you've described here sounds good for pre-built images users can directly download and flash.
